changes/1436.feature.rst

```diff
@@ -0,0 +1 @@
+Development: Migrated to use towncrier for change logs.
```

Comment:
add "See docs/development.rst for details and usage"

Reply:
DONE. Also in pywbem